components/kyma-environment-broker/internal/orchestration/manager/manager.go

```diff
@@ -88,9 +89,11 @@ func (m *orchestrationManager) resolveOperations(o *internal.Orchestration) ([]o
 		for _, r := range runtimes {
 			windowBegin := time.Time{}
 			windowEnd := time.Time{}
+
 			if o.Parameters.Strategy.Schedule == orchestration.MaintenanceWindow {
-				windowBegin, windowEnd = m.resolveWindowTime(r.MaintenanceWindowBegin, r.MaintenanceWindowEnd)
+				windowBegin, windowEnd = m.resolveWindowTime(r.MaintenanceWindowBegin, r.MaintenanceWindowEnd, r.MaintenanceDays)
```

Comment:
Here the plan is that `r.MaintenanceDays` will be filled by this point by `m.resolver.Resolve()` according to a specified maintenance window policy?